src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -24,6 +24,9 @@ public partial class DllNotFoundException : System.TypeLoadException
         public DllNotFoundException(string message) { }
         public DllNotFoundException(string message, System.Exception inner) { }
     }
+    public partial struct RuntimeArgumentHandle
```

Comment:
Github search really sucks for the large files like mscorlib.cs... but it is currently there https://github.com/dotnet/standard/blob/master/netstandard/ref/mscorlib.cs#L2817.